src/components/dialog/dialog.scss

```diff
@@ -45,13 +48,18 @@
     }
     .mdc-dialog__container {
         height: 100%;
+        width: var(--dialog-width, auto);
     }
 
     .mdc-dialog__surface {
         width: var(--dialog-width, auto);
         height: var(--dialog-height, auto);
         background-color: var(--dialog-background-color);
         box-shadow: var(--shadow-depth-64);
+
+        max-width: var(--dialog-max-width, calc(100% - 32px));
+        max-height: var(--dialog-max-height, calc(100% - 32px));
+        border-radius: var(--dialog-border-radius, 4px);
```

Comment:
```suggestion
        max-width: var(--dialog-max-width, calc(100% - 2rem));
        max-height: var(--dialog-max-height, calc(100% - 2rem));
        border-radius: var(--dialog-border-radius, 0.25rem);
```

Reply:
Fixed cbeb776
Also changed it to `100vw` instead of `100%` for the default max width since that was what Material was using. It might have been this that caused  the issues @FredrikWallstrom mentioned above

Reply:
Oh nice! that sounds better. I was thinking in terms of Portal and thought that 100% would be 100% of the viewport anyways when dialogs are opened in the page. But that's probably another story when it comes to this popup scenario.

Reply:
but I wonder if height should also be `100vh` in that case? 🤔

Reply:
It was not `100vh` in the Material defaults and I did not want to screw anything up :smile:

Reply:
I think it depends on the html structure of the popup. I don't know how to test this. But I when I think about it, it feels 100% should be alright and actually better, due to mobile phones.
[They say](https://developer.chrome.com/blog/url-bar-resizing/):
>  `vh` units will be sized to the viewport height as if the URL bar is always hidden. That is, `vh` units will be sized to the "largest possible viewport". This means `100vh` will be larger than the visible height when the URL bar is shown.

So the proper way to set the height to fill the visible viewport is with height: 100%, either on the <html> element or on a position: fixed element. and `.mdc-dialog` has `position: fixed`.